sg_otio/sg_cut_track_writer.py

```diff
@@ -837,7 +838,12 @@ def write_shots(self, clips_by_shots, sg_project, sg_linked_entity, sg_user=None
             sg_shots = self._sg.batch(sg_batch_data)
             # Update the clips Shots
             for sg_shot in sg_shots:
-                clip_group = clips_by_shots[sg_shot["code"]]
+                # clips_by_shots has the lowercased name as key.
+                clip_group = clips_by_shots.get(sg_shot["code"].lower())
```

Comment:
Why not just having `clips_by_shots[sg_shot["code"].lower()]`? It sounds that it must be there now?

Reply:
I preferred raising the error myself so that if it happens to a third party user, he doesn't see a `KeyError`.

Reply:
But it's not supposed to happen anymore, right?

Reply:
no, it's not, you're right. I can remove the `.get` if you feel it's better.

Reply:
yes please.